packages/schematics/angular/application/index_spec.ts

```diff
@@ -146,6 +146,30 @@ describe('Application Schematic', () => {
     expect(confContent.projects['foo-e2e']).toBeUndefined();
   });
 
+  it('should create correct files when using minimal', () => {
+    const options = { ...defaultOptions, minimal: true };
+    const tree = schematicRunner.runSchematic('application', options, workspaceTree);
+    const files = tree.files;
+
+    expect(files.indexOf('/projects/foo/karma.conf.js')).toBe(-1);
```

Comment:
Please use `toContain()` or `.not.toContain()` instead of looking for indices...

Reply:
@hansl sry about that, I'll update this and a couple of them in a seperate PR.